sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -19,6 +19,7 @@ package org.apache.spark.sql
 
```

Comment:
This file should be renamed to Dataset.scala. Didn't do this yet for better diff view.